pandas/core/internals/blocks.py

```diff
@@ -284,16 +283,10 @@ def __repr__(self) -> str:
         # don't want to print out all of the items here
         name = type(self).__name__
         if self._is_single_block:
-
             result = f"{name}: {len(self)} dtype: {self.dtype}"
-
         else:
-
-            shape = " x ".join(pprint_thing(s) for s in self.shape)
-            result = (
-                f"{name}: {pprint_thing(self.mgr_locs.indexer)}, "
```

Comment:
AFICT the ```pprint_thing``` is just a python2 thing.

@jbrockmendel can you please confirm or explain what it is?

Reply:
Yes - shouldn’t be needed now py3 only. Worth checking though

Reply:
> can you please confirm or explain what it is?

I think it might matter if you were dealing with nested objects, but for these i think it is just a leftover py2 thing